src/mmw/js/src/modeling/models.js

```diff
@@ -103,8 +136,8 @@ var ProjectModel = Backbone.Model.extend({
 
 var ModificationModel = coreModels.GeoModel.extend({
     defaults: _.extend({
-            name: '',
-            type: ''
+            'name': '',
+            'type': ''
```

Comment:
Ah, that's actually a mistake. `type` should be renamed to `value` I just didn't get around to fixing it yet.